### DISCUSSION TITLE
Help with this error: name 'getoutput' is not defined

### DISCUSSION THREAD

**Discussion by zamakhtar** [TheLastBen/fast-stable-diffusion#287] [Q&A]
![awefaf](https://user-images.githubusercontent.com/116928134/198841998-f2c886cf-1830-46c4-bda2-8f872786b39d.JPG)

Anyone have any idea how to fix this error? Happens when I try to start stable diffusion. It was working yesterday, though, so I'm not sure what is causing it.

**Comment by TheLastBen** [comment]
did you run all the cells from the top ?
